packages/ffe-searchable-dropdown-react/less/ffe-multiselect-searchable-dropdown.less

```diff
@@ -0,0 +1,203 @@
+        flex-wrap: wrap;
+        gap: var(--ffe-spacing-xs);
+        position: relative;
+        width: 100%;
+        min-height: 2.8125rem;
+        padding: var(--ffe-spacing-xs);
+        font-family: var(--ffe-g-font);
+        font-variant-numeric: tabular-nums;
+        background-color: var(--ffe-v-input-bg-color);
+        color: var(--ffe-v-input-color);
+        border-radius: var(--ffe-g-border-radius) 0 0 var(--ffe-g-border-radius);
+        border: 2px solid var(--ffe-g-border-color);
+        border-right: none;
+        transition: all var(--ffe-transition-duration) var(--ffe-ease);
+        font-size: var(--ffe-fontsize-form-input);
+    }
+
+    &--input-field {
+        border: none;
+        flex: 1;
+        margin: 0 0 0 var(--ffe-spacing-xs);
+        min-width: 10ch;
+        padding: 0;
+        width: 100%;
+        background-color: transparent;
+        color: var(--ffe-v-input-color);
+
+        &:focus,
+        &:active {
+            outline: none;
+        }
+    }
+
+    &__button {
+        background: transparent;
+        height: auto;
+        border: 2px solid var(--ffe-g-border-color);
+        border-left: none;
+        width: fit-content;
+        padding: 0 var(--ffe-spacing-xs);
+        box-shadow: none;
+        outline: none;
+        cursor: pointer;
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        border-radius: 0 var(--ffe-g-border-radius) var(--ffe-g-border-radius) 0;
+        transition: border var(--ffe-transition-duration) var(--ffe-ease), border-left 0s;
+        color: var(--ffe-g-primary-color);
+
+        > .ffe-icons {
+            color: var(--ffe-g-primary-color);
+        }
+
+        &:focus {
+            border: 2px solid var(--ffe-g-primary-color);
+        }
+
+        &-icon {
+            transition: color var(--ffe-transition-duration) var(--ffe-ease),
+            transform var(--ffe-transition-duration) var(--ffe-ease);
+        }
+
+
+        &--flip {
+            .ffe-searchable-dropdown__button-icon {
+                transform: rotateZ(180deg);
+            }
+        }
+    }
+
+    &__list-item-container[aria-selected='true'] {
+        background-color: var(--ffe-v-searchable-dropdown-multiselect-background-color-highlighted);
+    }
+
+    &__list-item-container:not(:last-child) {
+        border-bottom: 1px solid var(--ffe-g-border-color);
+    }
+
+    &__list--post-list-element {
+        border-top: 1px solid var(--ffe-g-border-color);
+        color: var(--ffe-g-text-color);
+    }
+
+    &__list-item-body {
+        display: grid;
+        grid-column-gap: var(--ffe-spacing-sm);
+        grid-template-columns: 1fr @check-icon-width;
+        align-items: center;
+        padding: var(--ffe-spacing-sm);
+        cursor: pointer;
+
+        &--condensed {
+            padding-block: var(--ffe-spacing-xs);
+        }
+
+        @media (hover: hover) and (pointer: fine) {
+            &:hover, &:focus, &:active {
```

Comment:
her vill focus og active kun slå in hvis man har muspeker. Dette active vill alltså ikke funke på mobil.

Reply:
Det går kanskje greit? Ikke sikker på hvordan det skal se ut på mobil